starboard/common/test_main.cc

```diff
@@ -27,12 +27,19 @@ int InitAndRunAllTests(int argc, char** argv) {
 }
 }  // namespace
 
-// When we are building Evergreen we need to export SbEventHandle so that the
-// ELF loader can find and invoke it.
+#if BUILDFLAG(IS_STARBOARD)
+// For the Starboard OS define SbEventHandle as the entry point
 SB_EXPORT STARBOARD_WRAP_SIMPLE_MAIN(InitAndRunAllTests);
+
+#if !SB_IS(EVERGREEN)
+// Define main for non-Evergreen Starboard OS.
```

Comment:
micro-nit: `main()` since it refers to a function.

Reply:
Done